<!DOCTYPE html>
<html lang="en">
<!-- Beautiful Jekyll 5.0.0 | Copyright Dean Attali 2020 -->
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  

  

  <title>Research</title>

  
  <meta name="author" content="Lorenzo Calviello">
  

  <meta name="description" content="The main interests of the lab can be divided into 4 main areas: ### RNP dynamics along the mRNA during normal and misregulated translation Tight control of cytoplasmic gene expression is orchestrated by the action of multiple ribonucleoprotein (RNP) complexes. One prominent example is the human ribosome, which undergoes drastic...">

  

  

  <link rel="alternate" type="application/rss+xml" title="Calviello Lab (beta version)" href="http://localhost:4000/feed.xml">

  

  

  


  
    
      
  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">


    
      
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css">


    
      
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic">


    
      
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800">


    
  

  
    
      <link rel="stylesheet" href="/assets/css/bootstrap-social.css">
    
      <link rel="stylesheet" href="/assets/css/beautifuljekyll.css">
    
  

  

  
  
  

  

  
  <meta property="og:site_name" content="Calviello Lab (beta version)">
  <meta property="og:title" content="Research">
  <meta property="og:description" content="The main interests of the lab can be divided into 4 main areas: ### RNP dynamics along the mRNA during normal and misregulated translation Tight control of cytoplasmic gene expression is orchestrated by the action of multiple ribonucleoprotein (RNP) complexes. One prominent example is the human ribosome, which undergoes drastic...">

  

  
  <meta property="og:type" content="website">
  <meta property="og:url" content="http://localhost:4000/research/">
  <link rel="canonical" href="http://localhost:4000/research/">
  

  
  <meta name="twitter:card" content="summary">
  
  <meta name="twitter:site" content="@">
  <meta name="twitter:creator" content="@">

  <meta property="twitter:title" content="Research">
  <meta property="twitter:description" content="The main interests of the lab can be divided into 4 main areas: ### RNP dynamics along the mRNA during normal and misregulated translation Tight control of cytoplasmic gene expression is orchestrated by the action of multiple ribonucleoprotein (RNP) complexes. One prominent example is the human ribosome, which undergoes drastic...">

  

  


  

  

</head>


<body>

  


  <nav class="navbar navbar-expand-xl navbar-light fixed-top navbar-custom top-nav-regular"><a class="navbar-brand" href="http://localhost:4000/">Calviello Lab (beta version)</a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#main-navbar" aria-controls="main-navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>

  <div class="collapse navbar-collapse" id="main-navbar">
    <ul class="navbar-nav ml-auto">
          <li class="nav-item">
            <a class="nav-link" href="/research">Research</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="/team">Team</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="/news">News</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="/publications">Publications</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="/softdata">Software & Data</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="/contact">Contact</a>
          </li></ul>
  </div>

  

  

</nav>


  <!-- TODO this file has become a mess, refactor it -->







<header class="header-section ">

<div class="intro-header no-img">
  <div class="container-md">
    <div class="row">
      <div class="col-xl-8 offset-xl-2 col-lg-10 offset-lg-1">
        <div class="page-heading">
          <h1>Research</h1>
          

          
        </div>
      </div>
    </div>
  </div>
</div>
</header>





<div class=" container-md " role="main">
  <div class="row">
    <div class=" col-xl-12 offset-xl-0 col-lg-12 offset-lg-0 ">
      

      <p>The main interests of the lab can be divided into 4 main areas:</p>

<h3 id="rnp-dynamics-along-the-mrna-during-normal-and-misregulated-translation">RNP dynamics along the mRNA during normal and misregulated translation</h3>

<p class="text-justify">Tight control of cytoplasmic gene expression is orchestrated by the action of multiple ribonucleoprotein (RNP) 
complexes. One prominent example is the human ribosome, which undergoes drastic changes as it scans through the mRNA and engages in protein synthesis.
In recent years, translation profiling (with techniques such as ribosome profiling, or Ribo-seq) has exposed a previously under-appreciated complexity in translation with incredible resolution, revealing active translation on the non-coding transcriptome,
and allowing for signal processing investigation of ribosome dynamics.</p>
<p align="center">
<img src="/img/ribot.png" />
</p>
<p class="text-justify">Moreover, analysis of protein-RNA complexes revealed an unexpected diversity to the ribosomal RNP itself, with hundreds of factors interacting with the ribosome throughout the translation cycle.
Despite an increasing amount of protein-RNA footprinting datasets, our ability to characterize binding and functions of heterogeneous ribosomal complexes along the transcriptome remains limited.</p>
<p align="center">
<img src="/img/rnp_dyn_ok.png" />
</p>
<p class="text-justify">Projects in this area aim at quantifying the interplay between functions of heterogeneous RNPs and RNA sequence-structure features, using experimental methods aimed at selective profiling of different RNP complexes, and computational analysis of transcriptome-wide tracks aided by signal processing and deep learning techniques. In addition, we aim at robustly detecting alternative modes of translation initiation and elongation, often ignored by standard analysis methods.</p>
<p><br /></p>

<h3 id="translation-in-gene-expression-control-across-cell-types-and-states">Translation in gene expression control across cell types and states</h3>

<p class="text-justify">RNA cis-regulatory elements and RBP binding can modulate the rate of protein synthesis on the mRNA, providing an additional mode of gene expression control. Moreover, decades of detailed molecular investigations in protein synthesis have uncovered multiple connections between nascent RNA processing, translation, and mRNA decay. Despite the vast amounts of data depicting these phenomena transcriptome-wide and with great accuracy, an analytical framework able to quantify and predict regulation at different steps of the gene expression cascade is missing.</p>
<p align="center">
<img src="/img/regul3.png" />
</p>
<p class="text-justify">Projects in this area will make extensive use of thousands of available Ribo-seq and RNA-seq datasets, controlled in silico experiments and machine learning to quantify different modes of gene expression regulation in multiple cell types and under different perturbations. In addition, more targeted studies in this area will focus on elucidating the molecular functions of RNA binding proteins, subcellular regulatory processes, and regulation by signaling pathways like mTOR and the Integrated Stress Response.</p>
<p><br /></p>

<h3 id="alternative-rna-processing-and-its-contribution-to-protein-synthesis-and-cell-physiology">Alternative RNA processing and its contribution to protein synthesis and cell physiology</h3>

<p class="text-justify">RNA-sequencing (RNA-seq) data has enabled researchers to quantify expression of thousands of genes, including canonical example of protein coding genes to more cryptic transcripts of unknown function; moreover, different transcript isoforms from the same genes have often been shown to exhibit tissue-specific functions, highlighting the need of transcript-level investigation into gene regulation. For hundreds of genes, computational analysis of Ribo-seq data has provided evidence for translation and also co-translational mRNA decay in a transcript-specific fashion, providing a powerful window into the cytoplasmic fate of thousands of transcripts. However, the extent of transcript-specific translational control, and the physiological relevance for the presence of multiple translated isoforms remains elusive.</p>
<p align="center">
<img src="/img/altspl.png" />
</p>
<p class="text-justify">Projects in this area will aim at quantifying the contribution of annotated and novel transcripts to the ensemble of translated sequences, using experimental methods measuring translation and RNA decay (with a focus on long-read technologies), and computational strategies to identify novel transcripts and deconvolve signals coming from multiple isoforms.</p>
<p><br /></p>

<p>In addition, in collaboration with the Neurogenomics department, we will investigate how the interplay of alternative splicing, co-translational RNA surveillance and localized translation
regulation shapes neuronal functions and underlies several neuronal pathologies.</p>

<p align="center">
<img src="/img/neur2.png" />
</p>

<h3 id="proteogenomics-applications-between-rna-and-protein">Proteogenomics applications between RNA and protein</h3>

<p class="text-justify">RNA sequencing technologies are an essential way to detect expressed transcripts, thus providing useful priors to identify synthesized proteins. As shown by different studies, careful analysis of Ribo-seq profiles can reveal protein synthesis events with high confidence and aid the discovery of new proteins being translated from uORFs, lincRNAs and alternative transcript isoforms. While earlier studies took advantage of proteomics data to qualitatively validate protein synthesis at uncharacterized loci, re-analysis and quantification of mass-spectrometry datasets revealed strong quantitative agreements between translation output and steady-state protein abundance. Despite these recent promising results, integrating transcriptomics with the complex world of proteome dynamics, considering regulation via post-translational modification, investigation of protein isoforms, and protein subcellular localization and function, remains an unmet challenge in the post-genomics era. 
In addition, a complex interplay between translation elongation dynamics and co-translational protein folding ensure correct protein stability and function, confirming the importance of jointly investigating RNP dynamics and protein activity.</p>
<p align="center">
<img src="/img/proteogen.png" />
</p>
<p class="text-justify">Projects in this area will analyze mass spectrometry data aided by RNA-seq and Ribo-seq analysis, with a focus on detecting and quantifying alternative proteins from translation of multiple transcripts isoforms, novel ORFs with unclear coding potential, and variations of “canonical” ORFs. In addition, we would like to initiate collaborative proteogenomics initiatives with the larger scientific community, to generate joint projects aimed at bridging next-generation sequencing and shotgun proteomics technologies.</p>
<p><br /></p>



      

      

    </div>
  </div>
</div>


  <footer>
  <div class="container-md beautiful-jekyll-footer">
    <div class="row">
      <div class="col-xl-8 offset-xl-2 col-lg-10 offset-lg-1">
      <ul class="list-inline text-center footer-links"><li class="list-inline-item">
    <a href="mailto:calviello.l.bio@gmail.com" title="Email me">
      <span class="fa-stack fa-lg" aria-hidden="true">
        <i class="fas fa-circle fa-stack-2x"></i>
        <i class="fas fa-envelope fa-stack-1x fa-inverse"></i>
      </span>
      <span class="sr-only">Email me</span>
   </a>
  </li><li class="list-inline-item">
    <a href="https://github.com/calviellolab" title="GitHub">
      <span class="fa-stack fa-lg" aria-hidden="true">
        <i class="fas fa-circle fa-stack-2x"></i>
        <i class="fab fa-github fa-stack-1x fa-inverse"></i>
      </span>
      <span class="sr-only">GitHub</span>
   </a>
  </li><li class="list-inline-item">
   <a href="https://orcid.org/0000-0002-5600-0988" title="ORCID">
     <span class="fa-stack fa-lg" aria-hidden="true">
       <i class="fas fa-circle fa-stack-2x"></i>
       <i class="fab fa-orcid fa-stack-1x fa-inverse"></i>
     </span>
     <span class="sr-only">ORCID</span>
   </a>
 </li><li class="list-inline-item">
    <a href="https://scholar.google.com/citations?user=p2emlPUAAAAJ&hl=en" title="Google Scholar">
      <span class="fa-stack fa-lg" aria-hidden="true">
        <i class="fas fa-circle fa-stack-2x"></i>
        <i class="fa fa-graduation-cap fa-stack-1x fa-inverse"></i>
      </span>
      <span class="sr-only">Google Scholar</span>
    </a>
  </li></ul>

      
      <p class="copyright text-muted">
      
        Lorenzo Calviello
        &nbsp;&bull;&nbsp;
      
      2021

      

      
      </p>
      <p class="theme-by text-muted">
        Powered by
        <a href="https://beautifuljekyll.com">Beautiful Jekyll</a>
      </p>
      </div>
    </div>
  </div>
</footer>


  
  
    
  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha256-4+XzXVhsDmqanXGHaHvgh1gMQKX40OUvDEBTu8JcmNs=" crossorigin="anonymous"></script>


  
    
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>


  
    
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>


  



  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
      <script src="/assets/js/beautifuljekyll.js"></script>
    
  









</body>
</html>
